spec/requests/api/public/v1/providers/index_spec.rb

```diff
@@ -0,0 +1,122 @@
+require "rails_helper"
+
+describe "GET public/v1/recruitment_cycle/:recruitment_cycle_year/providers" do
+  let(:organisation) { create(:organisation) }
+  let(:recruitment_cycle) { find_or_create :recruitment_cycle }
+
+  let(:provider) {
+    create(:provider,
+           provider_code: "1AT",
+           provider_name: "First provider",
+           organisations: [organisation],
+           contacts: [contact])
+  }
+
+  let(:contact) { build(:contact) }
+
+  let(:json_response) { JSON.parse(response.body) }
+
+  def perform_request
+    get request_path
+  end
+
+  subject do
+    perform_request
+
+    response
+  end
+
+  context "Searching for a provider" do
```

Comment:
https://api.qa.publish-teacher-training-courses.service.gov.uk/api-reference.html#provider_suggestions-get

What's the overriding reason over using the other endpoint?

Reply:
Actually that is true. I'll use that instead and close this PR